setup.py

```diff
@@ -50,7 +50,7 @@
         'testing': [
             'coverage', 'pytest', 'yapf==0.31', 'pytest-cov', 'transformers',
             'pytest-xdist', 'torchvision', 'tabulate', 'efficientnet_pytorch',
-            'pytest-timeout'
+            'pytest-timeout', 'pytorch_geometric'
```

Comment:
That's not the right line. https://pytorch-geometric.readthedocs.io/en/latest/notes/installation.html

Reply:
Changed it to the correct dependency but I'm not sure how to test it correctly.